.gitignore

```diff
@@ -50,6 +50,9 @@ tests/scripts/core/*.f90
 tests/scripts/core/*.mod
 tests/scripts/core/*.o
 
+tests/pyccel/scripts/headers.pyh
+tests/pyccel/scripts/test_headers.py
```

Comment:
why you ignore these files?

Reply:
we discussed about it in the link below
https://github.com/pyccel/pyccel/pull/574#discussion_r539303729